### PR TITLE
feat: added faq section

### PR DESCRIPTION
**Pull request by praptippradhan**

### CODE REVIEW

File: src/components/Gallery/gallery.jsx
Comment (jayeshxn) [inline src/components/Gallery/gallery.jsx:1]: make different PR for this component

File: src/components/Gallery/gallery.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+				{galleryData.images.map(({ id, src, alt }) => (
+					<div
+						key={id}
+						style={{
+							borderRadius: '4.5px',
+							border: '0.75px solid #000',
+							padding: '10.2px 9px 55.3px 9px',
+							background: '#FFF',
+							boxShadow: '1.5px 2.25px 0px 0px #000',
+						}}>
```
Comment (jayeshxn) [inline src/components/Gallery/gallery.jsx:28]: change to tailwind css

File: src/components/Gallery/galleryMain.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+function GalleryMain() {
+	return (
+		<div className='w-[90%] mx-auto flex flex-col items-center mb-[40px] gap-y-[105px]'>
+			<Heading
+				variant='h1'
+				style={{
+					fontSize: '80px',
+					color: '#F2DA05',
+					textShadow: '4.608px 4.608px 0px #000',
```
Comment (jayeshxn) [inline src/components/Gallery/galleryMain.jsx:11]: use tailwind

**Review by jayeshxn — CHANGES_REQUESTED** [review]
Please complete the requested changes.

**Comment by jayeshxn** [comment]
LGTM🚀

**Review by jayeshxn — APPROVED** [review]

**Comment by Shurtu-gal** [comment]
@all-contributors add @praptippradhan for code and add @jayeshxn for review.

**Comment by Shurtu-gal** [comment]
@all-contributors 
please add @praptippradhan for code
please add @jayeshxn for review.

**Pull request merged**